queue_services/entity-filer/src/entity_filer/common/enum.py

```diff
@@ -11,6 +11,7 @@ class BaseMeta(EnumMeta):
     def __contains__(self, other):
         """Return True if 'in' the Enum."""
         try:
+            # pylint: disable-next=no-value-for-parameter
             self(other)
```

Comment:
Disabling the lint check here since this is in a try-catch block.

The linter does not like that the variable other of type any is being passed as EnumMeta, and it throws the no-value-for-parameter error.